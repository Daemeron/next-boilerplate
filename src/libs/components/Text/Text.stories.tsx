import React, { Fragment } from 'react';

import { Text } from './Text';

export default {
  title: 'Components/Text',
};

export const WithText = (): JSX.Element => (
  <Fragment>
    <Text>dummy text</Text>
  </Fragment>
);
