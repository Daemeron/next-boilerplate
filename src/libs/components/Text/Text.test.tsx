import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import { Text } from './Text';

describe('Paragraph', () => {
  test('matches the snapshot', () => {
    const paragraph = render(<Text>test</Text>);
    expect(paragraph.container).toMatchSnapshot();
  });
});
