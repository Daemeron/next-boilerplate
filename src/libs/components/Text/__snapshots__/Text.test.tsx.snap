// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph matches the snapshot 1`] = `
<div>
  <p
    class=""
  >
    test
  </p>
</div>
`;
